pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -47,12 +47,9 @@ func defaultPredicates() sets.String {
 		predicates.MatchInterPodAffinityPred,
 		predicates.NoDiskConflictPred,
 		predicates.GeneralPred,
-		predicates.CheckNodeMemoryPressurePred,
-		predicates.CheckNodeDiskPressurePred,
-		predicates.CheckNodePIDPressurePred,
-		predicates.CheckNodeConditionPred,
 		predicates.PodToleratesNodeTaintsPred,
```

Comment:
I think `PodToleratesNodeTaintsPred` should be mandatory; if user enable TaintNodeByCondition without this predicates in configuration, will we change the behavior?

BTW, should we also identify mandatory predicates to make sure scheduler always "work".

Reply:
> I think PodToleratesNodeTaintsPred should be mandatory; if user enable TaintNodeByCondition without this predicates in configuration, will we change the behavior?

Ops, never mind; just see other diffs handled that.